### PR TITLE
RIUI-1724 Logger implemented 

### PR DESCRIPTION
**Pull request by dpanov76**

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+// isDebugMode from 0 to 3 when 0 then logger completely disabled
 
 export const environment = {
-  production: false
+  production: false,
+  isDebugMode: 3
```
Comment (matthewspeake-hmcts) [inline src/environments/environment.ts]: loggingLevel would be a better name for this.  isXXX implies a boolean.

Comment (matthewspeake-hmcts) [reply src/environments/environment.ts]: Also, is there a way of maintaining a map for a meaningful level to map what you showed yesterday:

1=ERROR
2=WARN
3=INFO
4=DEBUG

so we could set it like this:

```
loggingLevel: 'WARN'
```

or 

```
loggingLevel: Level.WARN
```

**Comment by matthewspeake-hmcts** [comment]
We can do those changes I mentioned in another ticket, for now it's good to go! 👍 Nice work! ;)

File: src/environments/environment.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+// isDebugMode - 'INFO' 'WARN 'ERROR'
 
 export const environment = {
```
Comment (frontr-uk) [inline src/environments/environment.ts]: create interface: 

export interfaces  ENVIROMENT {
 production: boolean,
isDebugMode: 'INFO', 'WARN', 'ERROR'
}

export const enviroment: ENVIROMENT {

}

Comment (frontr-uk) [reply src/environments/environment.ts]: this way we make sure the user does not accidentally put a wrong value in the property.

**Pull request merged**